templates/ssl.conf.j2

```diff
@@ -0,0 +1,8 @@
+[Service]
+{% if mesos_ssl_enabled %}
```

Comment:
I was not 100% sure if an empty service file is valid/allowed, hence I always kept the `[Service]` part in the file. I checked it on my system and it is not a problem.